dev/parm/config/gfs/config.fit2obs

```diff
@@ -11,6 +11,8 @@ echo "BEGIN: config.fit2obs"
 export PRVT=${FIXgfs}/gsi/prepobs_errtable.global
 export HYBLEVS=${FIXgfs}/am/global_hyblev.l${LEVS}.txt
 
+export COM_VRFYARCH="${ROTDIR}/vrfyarch"
```

Comment:
Will this be made `COMIN_VRFYARCH` later?

Reply:
Yes, thank you for catching my oversight @DavidHuber-NOAA ! I will change it to `COMIN_VRFYARCH` in the upcoming Fit2Obs task.